Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -137,6 +138,23 @@ class FunctionParser : public Parser<void>, public FunctionParserTypes<typename
     ControlStack& controlStack() { return m_controlStack; }
     Stack& expressionStack() { return m_expressionStack; }
 
+    void pushLocalInitialized(uint32_t index)
+    {
+        if (Options::useWebAssemblyTypedFunctionReferences() && !isDefaultableType(typeOfLocal(index)) && !localIsInitialized(index)) {
+            m_localInitStack.uncheckedAppend(index);
```

Comment:
Why is this unchecked?

Reply:
In `parse()` (https://github.com/WebKit/WebKit/blob/56633a9f762503fd20ea2be164489d70b9cf4083/Source/JavaScriptCore/wasm/WasmFunctionParser.h#L362) the `m_localInitStack` is sized to be the number of non-defaultable locals. Since we only ever push non-defaultable locals to this stack, it's not supposed to exceed that size.

Reply:
Let's just leave it as checked. Unchecked is only supposed to be for super perf critical things, or things where right above you can see why it doesn't need a check

Reply:
Ok no problem, will change it to normal checked `append(index)`.